Makefile

```diff
@@ -83,3 +83,14 @@ $(RPM_PACKAGE): | $(INSTALL_BIN)/$(NAME) install-config
 dev: gokeyless
 gokeyless: $(shell find . -type f -name '*.go')
 	go build -ldflags "-X main.version=dev" -o $@ ./cmd/gokeyless/...
+
+SOFTHSM_TOKENS_PATH          := $(DESTDIR)/var/lib/softhsm/tokens
+SOFTHSM_CONFIG_PATH          := $(DESTDIR)/etc
+
+.PHONY: dev-softhsm
+dev-softhsm:
+	@mkdir -p  $(SOFTHSM_CONFIG_PATH)
+	@mkdir -p  $(SOFTHSM_TOKENS_PATH)
+	@chmod 700 $(SOFTHSM_TOKENS_PATH)
+	@cp -r pkg/testing-tokens/*      $(SOFTHSM_TOKENS_PATH)
```

Comment:
The `pkg` folder for files which are included in the final deb/rpm packages, not test data. You should use the `testdata` folder for that. Using `$(DESTDIR)` doesn't make sense here either, as that's also for the deb/rpm packaging. The logic you have here makes sense, it just needs to use different folders rather than hijacking the deb/rpm build.

Reply:
But look in `install-config`, you're copying testing keys in the same place. How is this different? I'm just following that example.

Reply:
That's exactly my point - that target is used for building the deb/rpm packages.